test/bin/pyutils/build_bootc_images.py

```diff
@@ -111,78 +111,54 @@ def get_rhocp_beta_url_if_available(ver):
 
 
 def set_rpm_version_info_vars():
-    global SOURCE_VERSION
+    # See the test/bin/common_versions.sh script for a full list
+    # of the variables used for templating
     global MINOR_VERSION
     global PREVIOUS_MINOR_VERSION
     global YMINUS2_MINOR_VERSION
     global FAKE_NEXT_MINOR_VERSION
-    global SOURCE_VERSION_BASE
-    global CURRENT_RELEASE_VERSION
     global CURRENT_RELEASE_REPO
-    global PREVIOUS_RELEASE_VERSION
+    global CURRENT_RELEASE_VERSION
     global PREVIOUS_RELEASE_REPO
-    global RHOCP_MINOR_Y
-    global RHOCP_MINOR_Y1
-    global RHOCP_MINOR_Y2
-    global YMINUS2_RELEASE_VERSION
+    global PREVIOUS_RELEASE_VERSION
     global YMINUS2_RELEASE_REPO
+    global YMINUS2_RELEASE_VERSION
+    global RHOCP_MINOR_Y
     global RHOCP_MINOR_Y_BETA
+    global RHOCP_MINOR_Y1
     global RHOCP_MINOR_Y1_BETA
+    global RHOCP_MINOR_Y2
+
+    MINOR_VERSION = common.get_env_var('MINOR_VERSION')
+    PREVIOUS_MINOR_VERSION = common.get_env_var('PREVIOUS_MINOR_VERSION')
+    YMINUS2_MINOR_VERSION = common.get_env_var('YMINUS2_MINOR_VERSION')
+    FAKE_NEXT_MINOR_VERSION = common.get_env_var('FAKE_NEXT_MINOR_VERSION')
+    CURRENT_RELEASE_REPO = common.get_env_var('CURRENT_RELEASE_REPO')
+    CURRENT_RELEASE_VERSION = common.get_env_var('CURRENT_RELEASE_REPO')
+    PREVIOUS_RELEASE_REPO = common.get_env_var('PREVIOUS_RELEASE_REPO')
+    PREVIOUS_RELEASE_VERSION = common.get_env_var('PREVIOUS_RELEASE_VERSION')
+    YMINUS2_RELEASE_REPO = common.get_env_var('YMINUS2_RELEASE_REPO')
+    YMINUS2_RELEASE_VERSION = common.get_env_var('YMINUS2_RELEASE_VERSION')
+    RHOCP_MINOR_Y = common.get_env_var('RHOCP_MINOR_Y')
+    RHOCP_MINOR_Y_BETA = common.get_env_var('RHOCP_MINOR_Y_BETA')
+    RHOCP_MINOR_Y1 = common.get_env_var('RHOCP_MINOR_Y1')
+    RHOCP_MINOR_Y1_BETA = common.get_env_var('RHOCP_MINOR_Y1_BETA')
+    RHOCP_MINOR_Y2 = common.get_env_var('RHOCP_MINOR_Y2')
```

Comment:
Do we need those? I believe we use gomplate with .Env and I don't see these being used anywhere else.

Reply:
We do not need those atm, but I added them just in case to cover the full list. 
We're about to add upgrade tests to bootc, and there vars may be required.

Reply:
In which place do you foresee this variables being used? If gomplate, then it already has access to them, right?

Reply:
I agree. Let me remove unused variables.